### PR TITLE
Coding standards

### PR DESCRIPTION
**Pull request by evert**
Rebased #120

### CODE REVIEW

**Comment by DominikTo** [comment]
Rebased @evert's changes again. /cc @Hywan

**Comment by evert** [comment]
:+1:  yay

File: lib/Bin/AbstractCommand.php [hunk 1]
```diff
@@ -31,8 +31,7 @@
  * @author Ivan Enderlin
  * @license GNU Affero General Public License, Version 3.
  */
-abstract class AbstractCommand extends Console\Dispatcher\Kit
-{
+abstract class AbstractCommand extends Console\Dispatcher\Kit {
```
Comment (Hywan) [inline lib/Bin/AbstractCommand.php]: New line is missing.

File: lib/Bin/Install.php [hunk 2]
```diff
@@ -539,8 +530,7 @@ public function usage()
      *
      * @return string
      */
-    public static function getBaseURLInfo()
-    {
+    static function getBaseURLInfo() {
```
Comment (Hywan) [inline lib/Bin/Install.php]: Newline is missing.

**Comment by Hywan** [comment]
I will do another PR Monday for CS. As we said: PSR-1 + some extra stuff.

**Pull request merged**